### ISSUE TITLE
Highlight bindings implementing Copy

### ISSUE THREAD

**Issue by gilescope** [intellij-rust/intellij-rust#2029]
Would it be possible to colour or highlight differently literals and things that implement the Copy trait versus non-copyable things? It would be great to try this out as a proof of concept to see if it helped people. Is it possible / easy to try it?

**Comment by matklad** [comment]
Yeah, I think it should be not very hard to do: there'll be false positives, but they don't matter for highlighting a lot.
